internal/app/encryption.go

```diff
@@ -29,24 +32,13 @@ func FindIndex(vs []string, t string) int {
 
 // Password ..
 func Password() string {
-	mathrand.Seed(time.Now().UnixNano())
-	chars := []rune("ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
-		"abcdefghijklmnopqrstuvwxyz" +
-		"0123456789" +
-		"=+%*/()[]{}/!@#$?|")
+
 	length := viper.GetInt("server.generatedPasswordLength")
-	if length < 4 {
-		length = 4
-	}
-	var b strings.Builder
-	for i := 0; i < length; i++ {
-		randomNumber, err := rand.Int(rand.Reader, big.NewInt(int64(len(chars))))
-		if err != nil { // Fallback if something bad happens
-			randomNumber = big.NewInt(int64(mathrand.Intn(len(chars))))
-		}
-		b.WriteRune(chars[randomNumber.Int64()])
+	res, err := password.Generate(length, 10, 10, false, false)
```

Comment:
Not sure whether the length is required or not since  the issue #111 is saying default values good for us. But do not wanted to change it.

Reply:
No problem, this is good I think. Thank you.

Reply:
```suggestion
	func GeneratePassword() (string, error) {
              length := viper.GetInt("server.generatedPasswordLength")
               res, err := password.Generate(length, 10, 10, false, false) 
               if err != nil {
                      return "", err 
                }
               return res, nil 
        }